README.md

```diff
@@ -273,7 +282,18 @@ $ pie install xdebug --not-a-defined-configuration-option # this would fail
 
 ### Windows installation
 
-Determine the expected name for the Windows DLL:
+When installing the extension on Windows, the PHP version invoking PIE would be used, unless `--with-php-path=<php-path>` is
```

Comment:
"would be used as path to check the needed version, and install path" maybe? It's not going to be used for *running* PIE.

Reply:
Fixed, thank you!